docs/faq.md

```diff
@@ -9,11 +9,6 @@
   for Splunk products, access to other open-source projects including
   Micrometer. Note, we take an upstream-first approach, Splunk Distribution of
   OpenTelemetry Java allow us to move fast.
-- **Why don't you publish the javaagent jar to a Maven repository?** It would
-  make it very easy to accidentally put the agent on the application runtime
-  classpath, which may cause all sorts of problems and confusion - and the
-  agent won't work anyway, because it has to be passed in the `-javaagent` JVM
-  parameter.
```

Comment:
I guess we removed this because we _do_ publish to maven now. Might be nice to swizzle this around to say something like `Q: Should I put the agent on my classpath? A: No, the agent should never be on the runtime classpath`. Down the road, we could provide some gradle/mvn snippets that show how to source it without including it in the classpath....I dunno.

Reply:
Good point, will add.